.taskcluster.yml

```diff
@@ -0,0 +1,71 @@
+version: 1
+reporting: checks-v1
+policy:
+  pullRequests: public
+tasks:
+  $let:
+    head_rev:
+      $if: tasks_for == "github-pull-request"
+      then: ${event.pull_request.head.sha}
+      else: ${event.after}
+    repository:
+      $if: tasks_for == "github-pull-request"
+      then: ${event.pull_request.head.repo.clone_url}
+      else: ${event.repository.url}
+  in:
+    $let:
+      tests:
+        - name: lint-ruby
+          image: ruby
+          command:
+            - /bin/bash
+            - '--login'
+            - '-c'
+            - >-
+              git clone ${repository} repo && cd repo &&
+              git config advice.detachedHead false &&
+              git checkout ${head_rev} &&
+              mv .gemrc ~/.gemrc &&
+              gem install rubocop &&
+              rubocop
+        - name: brew {audit, install}
+          image: ruby:2.6
+          command:
+            - /bin/bash
+            - '--login'
+            - '-c'
+            - >-
+              apt-get update && apt-get install -y build-essential procps file sudo &&
+              adduser --disabled-password --gecos "" linuxbrew &&
+              adduser linuxbrew sudo &&
+              su - linuxbrew -c "NONINTERACTIVE=1 curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh | bash" &&
+              echo 'eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"' >> /home/linuxbrew/.profile &&
+              eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)" &&
+              git clone ${repository} repo && cd repo &&
+              git config advice.detachedHead false &&
+              git checkout ${head_rev} &&
+              brew audit --strict ./Formula/taskcluster.rb &&
+              brew install ./Formula/taskcluster.rb
```

Comment:
maybe it will be good to call the installed binary in the end? `taskcluster version` or so 

Btw, noticed in [logs](https://community-tc.services.mozilla.com/tasks/Lc3a4PBbRqGo46J4bYf_Hg/runs/0/logs/public/logs/live.log) for the install command:

```
Error: Failed to load cask: ./Formula/taskcluster.rb
Cask 'taskcluster' is unreadable: wrong constant name #<Class:0x000000000293cf90>
Warning: Treating ./Formula/taskcluster.rb as a formula.
==> Downloading https://github.com/taskcluster/taskcluster/releases/download/v44
```

Reply:
Looks like brew wants to initially treat the file as a cask, then falls back to formula. So I'll update this command to include the `--formula` flag so it doesn't print this error!